url/resources/toascii.json

```diff
@@ -102,6 +102,22 @@
     "input": "يa",
     "output": null
   },
+  {
+    "input": "\u064A.",
+    "output": "xn--mhb."
+  },
+  {
+    "input": "1.\u064A",
+    "output": "1.xn--mhb"
+  },
+  {
+    "input": "\u064A.1",
```

Comment:
This should be modified to have another label at the end as this will fail due to the IPv4 parser.